tokio/Cargo.toml

```diff
@@ -106,7 +106,7 @@ tracing = { version = "0.1.29", default-features = false, features = ["std"], op
 [target.'cfg(all(tokio_uring, target_os = "linux"))'.dependencies]
 io-uring = { version = "0.7.6", default-features = false }
 libc = { version = "0.2.168" }
-mio = { version = "1.0.1", default-features = false, features = ["os-poll", "os-ext"] }
+mio = { version = "1.0.4", default-features = false, features = ["os-poll", "os-ext"] }
```

Comment:
There's no need for this change. Tokio continues to be compatible with 1.0.1, 1.0.2, and 1.0.3.

Reply:
🤔 This would be good for forcing users to unify their `windows` crate ecosystem to the same versions, but I suppose it's not absolutely necessary to make things compile. Reverted with da52dfb1.